stan/math/prim/scal/fun/sum.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef STAN_MATH_PRIM_SCAL_FUN_SUM_HPP
+#define STAN_MATH_PRIM_SCAL_FUN_SUM_HPP
+
+namespace stan {
+namespace math {
+
+/**
+ * Acts as the identity function.
```

Comment:
Can you document this function differently?

It should say that it returns the sum as the summary. Then as a detail, it should say that this base case is for scalars and it will return the scalar value as the sum (your comment). As is, it makes it sound like the function `sum` is an identity function.